packages/block-editor/src/components/inner-blocks/index.native.js

```diff
@@ -106,6 +106,7 @@ class InnerBlocks extends Component {
 		const {
 			clientId,
 			renderAppender,
+			containerStyle,
```

Comment:
pass `containerStyle` from `Columns` to `BlockList`

Here I'm not sure if its en elegant solution according to architecture